From f8691b90f83c2058d82ac9b173b351ce3aa5b714 Mon Sep 17 00:00:00 2001
From: Anton Khirnov <anton@khirnov.net>
Date: Thu, 20 Jan 2022 09:18:31 +0100
Subject: [PATCH] Do not disable IPv6 on the physical interface for a VLAN
 port.

When one of the bridge ports is a VLAN interface, the ifupdown script
will currently disable IPv6 on the physical interface for that VLAN.
This is undesirable, since the user may want to use the physical
interface independently.
---
 debian/bridge-utils.sh | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/debian/bridge-utils.sh b/debian/bridge-utils.sh
index a3bfe5f..915c5d0 100644
--- a/debian/bridge-utils.sh
+++ b/debian/bridge-utils.sh
@@ -64,10 +64,6 @@ then
   # port is a vlan and the device exists?
   if [ "$port" != "$dev" ] && [ -e "/sys/class/net/$dev" ]
   then
-    if [ -f /proc/sys/net/ipv6/conf/$dev/disable_ipv6 ]
-    then
-      echo 1 > /proc/sys/net/ipv6/conf/$dev/disable_ipv6
-    fi
     ip link set "$dev" up
     ip link add link "$dev" name "$port" type vlan id "${port#*.}"
   fi
-- 
2.30.2

